catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/util/ProjectSetupUtils.java

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
since all of the functions in this class are very specifically designed for the two new tests, move the class over to /stage/ and rename to StageTestUtils. (eg. theres also a BrickTestUtils that also contains project setup stuff but again has different requirements than those for the stage)